umm-spec-lib/resources/json-schemas/variable/umm/v1.8/umm-var-1.8-sample.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
This file should go under the example_data directory and not with the schema.

Reply:
file removed
TL;DR
File deleted after manual inspection of test code and I did a grep for the file name resulting in no usage.